ci/occlum.sh

```diff
@@ -0,0 +1,10 @@
+rm -rf occlum_instance && mkdir occlum_instance && cd occlum_instance
+
+occlum init && rm -rf image
+
+# Set the kernel space heap size
+sed -i 's/\"kernel_space_heap_size\": \"32MB\"/\"kernel_space_heap_size\": \"100MB\"/g' Occlum.json;
```

Comment:
FWIW, I've found `jq` filters easier to manage.

Reply:
Great suggestion, let me fix this.